fabric/bin/lib/common_ledger.sh

```diff
@@ -48,7 +48,7 @@ define_common_vars() {
     : ${CHAN_ID:="mychannel"}
     : ${ERCC_ID:="ercc"}
     : ${ERCC_VERSION:="0"}
-    : ${SEQUENCE:="1"}
+    : ${ERCC_SEQUENCE:="1"}
```

Comment:
don't forget to change also the other references to SEQUENCE in peer.sh!
Maybe you are already in the process of doing so, in that case, try to not add a commit for every individual change but try to bundle related changes into a single commit.   You might also cross-check a bit more careful with a git diff/git diff -staged before you commit.  At least not sure whether your commit [33b875c2433fc5](https://github.com/hyperledger-labs/fabric-private-chaincode/pull/307/commits/33b875c2433fc53604551cd27c913dfe819e2a7c) was really intentional (the only thing it added was a single white-space difference comparted to the original version?)